pkg/apiserver/rest/usecase/helm.go

```diff
@@ -158,3 +183,12 @@ func flattenKey(prefix string, src map[string]interface{}, dest map[string]inter
 		}
 	}
 }
+
+func setAuthInfo(ctx context.Context, k8sClient client.Client, secretName string) (*common.HTTPOption, error) {
```

Comment:
```suggestion
func setAuthInfo(ctx context.Context, k8sClient client.Client, secretNameAndNamespace  types.NamespacedName) (*common.HTTPOption, error) {
```

Reply:
Advice to provide the namespace of the secret. For example, the addon registry could use any namespace to provide a secret.

Reply:
Reasonable. fixed